src/providers/user/user-provider.js

```diff
@@ -78,6 +78,12 @@ export default class UserProvider extends EventEmitter {
         return statusRoles.includes(role);
     }
 
+    async getPossibleRoles() {
+        const statusRoles = await this.roleStatus.getAllStatusRoles();
```

Comment:
Do the list of possible roles include non-status roles? If so, we may at least want to put a note about that here.

Reply:
You are correct, in this case only status roles are returned.